src/GovUk.Education.ExploreEducationStatistics.Admin/Services/Public.Data/PreviewTokenService.cs

```diff
@@ -148,6 +152,7 @@ private async Task<PreviewTokenViewModel> MapPreviewToken(PreviewToken previewTo
             Status = previewToken.Status,
             CreatedByEmail = createdByEmail,
             Created = previewToken.Created,
+            Activates = previewToken.Activates ?? previewToken.Created,
```

Comment:
See my other comment about possibly making `Activates` non-nullable, in which case this can be simplified.